src/components/bookslist.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
1. Let move this file into the `container` folder, for any of our component connecting to the redux store are kept inside the `container` folder.
![image](https://user-images.githubusercontent.com/31897434/74389667-3e925c80-4dff-11ea-8e0c-d533c73b0109.png)
2. we need to connect to the `redux store`
![image](https://user-images.githubusercontent.com/31897434/74389884-f293e780-4dff-11ea-89e8-3c5f82f8b448.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/